### DISCUSSION TITLE
中文标题/数据轴标签无法正确显示

### DISCUSSION THREAD

**Discussion by Roych13** [zai-org/ChatGLM3#496] [Bad Case]
能否通过Code Interpreter解决这个问题？目前，哪怕我在SYSTEM_PROMPT中加上类似于这样的提示：

> 当使用matplotlib绘图时，你允许添加以下语句来解决中文图表的乱码问题：
> plt.rcParams['font.sans-serif'] = ['God-FangSongGBK-free']

仍然无法在生成代码时给添加上去，暂时不清楚如何修改。

**Comment by zRzRzRzRzRzRzR** [comment]
> 能否通过Code Interpreter解决这个问题？目前，哪怕我在SYSTEM_PROMPT中加上类似于这样的提示：
> 
> > 当使用matplotlib绘图时，你允许添加以下语句来解决中文图表的乱码问题：
> > plt.rcParams['font.sans-serif'] = ['God-FangSongGBK-free']
> 
> 仍然无法在生成代码时给添加上去，暂时不清楚如何修改。

没有在训练集估计很难做，应该是解决不了的
